src/main/java/CommunicateCaseHandle.java

```diff
@@ -0,0 +1,134 @@
+import java.util.Scanner;
+public class CommunicateCaseHandle{
+    protected ExceptionsHandle e;
+    public CommunicateCaseHandle(){
+        e = new ExceptionsHandle();
+    }
+    public void dividingLine(){
+        System.out.println("\t__________________________________________________");
+    }
+    public void listHandle(int index, Task[] list){
```

Comment:
Leave a gap between the parameters and the open braces of the code

Reply:
THNAKS